lib/Common/CommonDefines.h

```diff
@@ -328,16 +328,24 @@
 #endif
 #endif // ENABLE_DEBUG_CONFIG_OPTIONS
 
-#if !defined(NTBUILD) || defined(ENABLE_DEBUG_CONFIG_OPTIONS)
 ////////
 //Time Travel flags
+#if !defined(NTBUILD) || defined(ENABLE_DEBUG_CONFIG_OPTIONS)
 #define ENABLE_TTD 1
```

Comment:
Lets add a comment here explaining that we want TTD to be enabled in fretest and chk build of SD and all flavors of VSO build